source/overview/security.rst

```diff
@@ -28,49 +28,59 @@ Private Cloud Deployment with Secure Mobile Apps
+   - Daily compliance reports compatible with third-party compliance solutions such as `Global Relay and Actiance <https://docs.mattermost.com/administration/compliance-export.html>`__ are also available (E20).
 
 Authentication Safeguards
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   - To protect against brute force attacks, you can set `rate limiting on APIs <https://docs.mattermost.com/administration/config-settings.html#id55>`__, varied by query frequency, memory store size, remote address and headers.
-   - Session length, session cache and idle timeout can be `configured according to your internal policies <https://docs.mattermost.com/administration/config-settings.html#sessions>`__, automatically forcing a user to re-login after a specified period of time.
-   - Remotely `revoke user sessions <https://docs.mattermost.com/help/settings/account-settings.html#view-and-logout-of-active-sessions>`__ across web, mobile devices and native desktop apps.
+   - To protect against brute force attacks, you can set `rate limiting on APIs <https://docs.mattermost.com/administration/config-settings.html#id55>`__, varied by query frequency, memory store size, remote address, and headers.
+   - Session length, session cache, and idle timeout can be `configured according to your internal policies <https://docs.mattermost.com/administration/config-settings.html#sessions>`__, automatically forcing a user to re-login after a specified period of time.
+   - Remotely `revoke user sessions <https://docs.mattermost.com/help/settings/account-settings.html#view-and-logout-of-active-sessions>`__ across web, mobile devices, and native desktop apps.
    - Remotely reset user passwords via the System Console or via the `command line <https://docs.mattermost.com/administration/command-line-tools.html#platform-user-password>`__.
    - Mattermost supports integrated authentication with `Active Directory and LDAP <https://docs.mattermost.com/deployment/sso-ldap.html>`__ (E10) as well as `SAML 2.0 SSO integration <https://docs.mattermost.com/deployment/sso-saml.html>`__ with providers including `Active Directory Federation Services <https://docs.mattermost.com/deployment/sso-saml-adfs.html>`__,  `Okta <https://docs.mattermost.com/deployment/sso-saml-okta.html>`__, among others (E20).
    - The ability to require `multi-factor authentication <https://docs.mattermost.com/deployment/auth.html>`__ is also available (E10).
 
 Access Control Policy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+If a user logs in and enters the correct email address but wrong password, a "your password is incorrect" response is presented. This can allow account enumeration. This is a deliberate design decision we've made for the product in the context of overall system security. 
```

Comment:
This has changed since this was written.  Lines 67-69 can be updated to state that we prevent account enumeration and provide a more generic login error when a user enters incorrect login information.

Reply:
Done, and reworded as per below comment.